ibm/service/kubernetes/data_source_ibm_container_cluster.go

```diff
@@ -426,7 +426,7 @@ func dataSourceIBMContainerClusterRead(d *schema.ResourceData, meta interface{})
 	}
 
 	albs, err := albsAPI.ListClusterALBs(name, targetEnv)
-	if err != nil && !strings.Contains(err.Error(), "The specified cluster is a lite cluster.") && !strings.Contains(err.Error(), "This operation is not supported for your cluster's version.") && !strings.Contains(err.Error(), "The specified cluster is a free cluster.") {
```

Comment:
So we don't have concept of any free cluster now. If some one provisioned free cluster with old version of provider will this n't effect

Reply:
There are no free clusters, once they were created, they had 30 days before they are automatically deleted,
So we will never hit that issue.